### ISSUE TITLE
Update pdxSearch.jsp

### ISSUE THREAD

**Issue by ahcurrier** [mgijax/mtb-wi#42]

**Comment by ahcurrier** [comment]
Ready for review here:
http://bhmtbdb01:8080/mtbwi2/pdxSearch.do

**Comment by DABegley** [comment]
When I look at the example page I get a 500 eror.

HTTP Status 500 - /WEB-INF/tags/searchform.tag (line: 10, column: 0) Attribute name invalid for tag form according to TLD

type Exception report

message /WEB-INF/tags/searchform.tag (line: 10, column: 0) Attribute name invalid for tag form according to TLD

description The server encountered an internal error that prevented it from fulfilling this request.

exception

org.apache.jasper.JasperException: /WEB-INF/tags/searchform.tag (line: 10, column: 0) Attribute name invalid for tag form according to TLD
	org.apache.jasper.compiler.DefaultErrorHandler.jspError(DefaultErrorHandler.java:42)
	org.apache.jasper.compiler.ErrorDispatcher.dispatch(ErrorDispatcher.java:443)
	org.apache.jasper.compiler.ErrorDispatcher.jspError(ErrorDispatcher.java:237)
	org.apache.jasper.compiler.Validator$ValidateVisitor.checkXmlAttributes(Validator.java:1266)
	org.apache.jasper.compiler.Validator$ValidateVisitor.visit(Validator.java:876)
	org.apache.jasper.compiler.Node$CustomTag.accept(Node.java:1553)
	org.apache.jasper.compiler.Node$Nodes.visit(Node.java:2392)
	org.apache.jasper.compiler.Node$Visitor.visitBody(Node.java:2444)
	org.apache.jasper.compiler.Node$Visitor.visit(Node.java:2450)
	org.apache.jasper.compiler.Node$Root.accept(Node.java:474)
	org.apache.jasper.compiler.Node$Nodes.visit(Node.java:2392)
	org.apache.jasper.compiler.Validator.validateExDirectives(Validator.java:1845)
	org.apache.jasper.compiler.Compiler.generateJava(Compiler.java:218)
	org.apache.jasper.compiler.Compiler.compile(Compiler.java:374)
	org.apache.jasper.compiler.Compiler.compile(Compiler.java:354)
	org.apache.jasper.compiler.Compiler.compile(Compiler.java:341)
	org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:662)
	org.apache.jasper.servlet.JspServletWrapper.loadTagFile(JspServletWrapper.java:247)
	org.apache.jasper.compiler.TagFileProcessor.loadTagFile(TagFileProcessor.java:577)
	org.apache.jasper.compiler.TagFileProcessor.access$000(TagFileProcessor.java:49)
	org.apache.jasper.compiler.TagFileProcessor$TagFileLoaderVisitor.visit(TagFileProcessor.java:654)
	org.apache.jasper.compiler.Node$CustomTag.accept(Node.java:1553)
	org.apache.jasper.compiler.Node$Nodes.visit(Node.java:2392)
	org.apache.jasper.compiler.Node$Visitor.visitBody(Node.java:2444)
	org.apache.jasper.compiler.Node$Visitor.visit(Node.java:2575)
	org.apache.jasper.compiler.Node$JspBody.accept(Node.java:2023)
	org.apache.jasper.compiler.Node$Nodes.visit(Node.java:2392)
	org.apache.jasper.compiler.Node$Visitor.visitBody(Node.java:2444)
	org.apache.jasper.compiler.TagFileProcessor$TagFileLoaderVisitor.visit(TagFileProcessor.java:658)
	org.apache.jasper.compiler.Node$CustomTag.accept(Node.java:1553)
	org.apache.jasper.compiler.Node$Nodes.visit(Node.java:2392)
	org.apache.jasper.compiler.Node$Visitor.visitBody(Node.java:2444)
	org.apache.jasper.compiler.Node$Visitor.visit(Node.java:2450)
	org.apache.jasper.compiler.Node$Root.accept(Node.java:474)
	org.apache.jasper.compiler.Node$Nodes.visit(Node.java:2392)
	org.apache.jasper.compiler.TagFileProcessor.loadTagFiles(TagFileProcessor.java:672)
	org.apache.jasper.compiler.Compiler.generateJava(Compiler.java:230)
	org.apache.jasper.compiler.Compiler.compile(Compiler.java:374)
	org.apache.jasper.compiler.Compiler.compile(Compiler.java:354)
	org.apache.jasper.compiler.Compiler.compile(Compiler.java:341)
	org.apache.jasper.JspCompilationContext.compile(JspCompilationContext.java:662)
	org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:364)
	org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:395)
	org.apache.jasper.servlet.JspServlet.service(JspServlet.java:339)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	org.apache.struts.action.RequestProcessor.doForward(RequestProcessor.java:1062)
	org.apache.struts.action.RequestProcessor.processForwardConfig(RequestProcessor.java:386)
	org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:229)
	org.apache.struts.action.ActionServlet.process(ActionServlet.java:1196)
	org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:414)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:624)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	org.apache.catalina.filters.CorsFilter.handleNonCORS(CorsFilter.java:436)
	org.apache.catalina.filters.CorsFilter.doFilter(CorsFilter.java:177)
	org.jax.mgi.mtb.webutils.filters.CompressionFilter.doFilter(CompressionFilter.java:171)
	org.jax.mgi.mtb.webutils.filters.TimerFilter.doFilter(TimerFilter.java:76)
note The full stack trace of the root cause is available in the Apache Tomcat/7.0.70 logs.

**Comment by DABegley** [comment]
Same 500 eror when I try to go to this page from the Searches/Tools pulldown menu.

**Comment by ahcurrier** [comment]
Thanks, please give it another try.

**Comment by CarolBult** [comment]
@ahcurrier 
I tried searching for bladder models with specific BRAF mutations and got a blank blue screen.
Maybe it is some setting on my browser but the text is still very small. The dialog boxes are a bit narrow and the spacing on the page in general is too generous. It needs to be more compact like the original search page. 


<img width="1054" alt="Screen Shot 2020-07-16 at 8 14 08 AM" src="https://user-images.githubusercontent.com/8211639/87669655-5f5b3700-c73c-11ea-89c5-4e29298736ac.png">

**Comment by ahcurrier** [comment]
Hi @CarolBult . Could you provide the exact search that produced this result? Is the compactness of the search form something that will prevent the new site from launching? Thanks.

**Comment by CarolBult** [comment]
@ahcurrier 
I select bladder as the cancer type and then ask for models that have mutations in BRAF. That results in the blue screen.
<img width="656" alt="Screen Shot 2020-07-16 at 9 33 48 AM" src="https://user-images.githubusercontent.com/8211639/87677332-a1d64100-c747-11ea-9bc6-a47b9908bceb.png">

**Comment by CarolBult** [comment]
As for the compactness issue..... We can launch w/o it but we'd need to do an update in short order. 

I see that you have tried to make the search parameters more compact. That section also needs to be labeled as "You searched for...."

**Comment by DABegley** [comment]
Works for me now, thanks.

**Comment by sneuhaus** [comment]
@ahcurrier I think this is fixed. I winged the change to the jax:dl tag. Please have a look. The old parameter genes returned a string not a collection.

**Comment by ahcurrier** [comment]
Parameters more compact, label updated, and gene info updated.
